### PR TITLE
Weekly report summary input

### PR DESCRIPTION
**Pull request by massproduction**

### CODE REVIEW

**Review by nakkje — CHANGES_REQUESTED** [review]
image
It's somewhat ugly, and IMO it's not super clear that it's possible to add n rows by saving between each

**Comment by massproduction** [comment]
@nakkje fixed the buttons.

**Comment by nakkje** [comment]
IMHO it would be much better if the summary was implemented as a WeeklyReportComment. Would get latest edits nicely too (and undo etc in the future if requested) 
I did it for change of scope thingy https://github.com/massproduction/solinor-invoice-checking/blob/weekly-report-change-of-scope/invoices/views.py#L736
https://github.com/massproduction/solinor-invoice-checking/blob/weekly-report-change-of-scope/invoices/views.py#L695
(that branch is still missing getting the latest change of scope 'comment' from previous weekly reports in case it's missing from current one, thus no pr yet)

**Comment by massproduction** [comment]
~~If it's just a single field then I don't see the point of implementing it as a different model.~~ Yeah this is needed for adding custom pages.

We still need the title (for the custom pages) to be in the WeeklyReportComment (is this a proper name?) table.

File: invoices/views.py [hunk 1]
```diff
@@ -734,3 +740,23 @@ def weekly_report_page(request, weekly_report_id, **_):
...
+
+    if request.POST.get("wr-summary"):
+        comment = WeeklyReportComments(type="S", text=request.POST.get("wr-summary"), user=request.user.email, weekly_report=weekly_report)
+        comment.save()
+
+    return HttpResponseRedirect(reverse("weekly_report", args=[weekly_report_id]))
+
+
+@login_required
+def delete_weekly_report_summary(request, weekly_report_id=None, weekly_report_comment_id=None):
```
Comment (nakkje) [inline invoices/views.py]: shouldn't it be delete_weekly_report_comment?

File: invoices/views.py [hunk 2]
```diff
@@ -734,3 +740,23 @@ def weekly_report_page(request, weekly_report_id, **_):
...
+    comment = get_object_or_404(WeeklyReportComments, id=weekly_report_comment_id)
+    if comment:
+        comment.delete()
```
Comment (nakkje) [inline invoices/views.py]: add message to request

**Review by nakkje — CHANGES_REQUESTED** [review]

**Review by nakkje — APPROVED** [review]

**Pull request merged**